apps/app-article/src/App.js

```diff
@@ -68,10 +68,22 @@ class App extends Component {
             this.setState({fontSize: parseFloat(localStorage.getItem("fontSize"))});
         }
 
+        if(getCookie('LoggedOutFlag')){
+            let d = new Date();
+            d.setTime(d.getTime() - (1000*60*60*24)); //Set the time to the past. 1000 milliseonds = 1 second
+            let expires = "expires=" + d.toUTCString();
```

Comment:
Also, should we move the cookie expiration inside the `onLogout` callback?

Reply:
Because it would be correct to do it after the logout occurs, not actually before.